packages/lisk-p2p/src/helpers.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { VError } from 'verror';
+import { Peer } from './peer';
+
+export interface PeerReturnType {
+	readonly options: PeerOptions;
+	readonly peers: ReadonlyArray<Peer>;
+}
+export interface PeerOptions {
+	readonly [key: string]: string | number;
+}
+/* tslint:disable: readonly-keyword*/
```

Comment:
`// tslint:disable-next-line` should work better

Reply:
`HistogramValues` also needs readonly keyword disabled, so I thought putting them in the block will look better as we will need multiple lines of `tslint:disable-next-line`

```
/* tslint:disable: readonly-keyword*/
interface Histogram {
	[key: number]: number;
}
interface HistogramValues {
	height: number;
	histogram: Histogram;
	max: number;
}
/* tslint:enable: readonly-keyword */
```